app/models/server/models/Rooms.js

```diff
@@ -21,6 +22,29 @@ export class Rooms extends Base {
 		this.tryEnsureIndex({ closedAt: 1 }, { sparse: true });
 	}
 
+	//
+	// Overriding insert method to create the genesis event
+	//
+	insert(...args) {
+		let [room] = args;
+
+		const roomId = super.insert(...args);
```

Comment:
```suggestion
		const roomId = super.insert(room, ...args);
```

Reply:
I can't change this, the parent signature (Base) is generic